docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -112,21 +117,28 @@ To close cases when they are sent to an external system, select
 *Automatically close Security cases when pushing new incident to external system*.
 
 [float]
-=== Change and update connectors
+=== Change the default connector
 
-You can create additional connectors, update existing connectors, and change
-the connector used to send cases to external systems.
+To change the default connector used to send cases to external systems, go to *Cases* -> *Edit external connection* and select the required connector from the *Incident management system list*.
 
 TIP: You can also configure which connector is used for each case individually
 (see <<cases-ui-open>>).
 
-. To change the default connector used to send cases to external systems:
-.. Go to *Cases* -> *Edit external connection*.
-.. Select the required connector from the *Incident management system* list.
-. To update an existing connector:
-.. Click *Update <connector name>*.
-.. Update the connector fields as required.
+[role="screenshot"]
+image::images/cases-change-default-connector.png[]
+
+[float]
+=== Modify connector settings
+
+To change the settings of an existing connector:
 
+. Go to *Cases* -> *Edit external connection*.
+. Select the required connector from the *Incident management system list*.
+. Click *Update <connector name>*.
+. From the Edit connector flyout, modify the connector fields as required and save the changes.
```

Comment:
Actually would like to poll the writing team for the proper preposition for "flyout" - I could be wrong here. 

```suggestion
. In the Edit connector flyout, modify the connector fields as required, then click *Save & close* to save your changes.
```
For the image beneath, crop out the background shadow part - just need to show the flyout.